__tests__/integration.js

```diff
@@ -0,0 +1,391 @@
+const awsServerlessExpress = require('../index')
+const apiGatewayEvent = require('../example/api-gateway-event.json')
+const app = require('../example/app')
+
+const server = awsServerlessExpress.createServer(app)
+const lambdaFunction = {
+  handler: (event, context) => awsServerlessExpress.proxy(server, event, context)
+}
+
+function clone(json) {
+  return JSON.parse(JSON.stringify(json))
+}
+
+function makeEvent(eventOverrides) {
```

Comment:
Helper function for making `event`s to reduce boilerplate. It uses the example event included in the `example` directory as a base and modifies it according to the needs of each test.